### PR TITLE
Runaway Birds

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## About The Pull Request
Jbird's Damage is decreased to 45.
Adds runaway birds that follow Jbird.
Jbird spawns 3 birds on spawn and 2 birds for every ding kill. These birds act as pale simple smiles, knocking you down and letting Jbird get a kill.

## Why It's Good For The Game
Jbird too easy. Should be harder than KOG.

## Changelog
:cl:
add: Added runaway birds to Jbird
balance: Jbird deals less damage
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/abnormality/waw/judgement_bird.dm [hunk 1]
```diff
@@ -103,3 +112,63 @@
...
+	..()
+	var/mob/living/simple_animal/hostile/runawaybird/V = new(get_turf(src))
+	birdlist+=V
+	V = new(get_turf(src))
+	birdlist+=V
+	V = new(get_turf(src))
+	birdlist+=V
+
+
+//Runaway birds - Mini Simple Smile, 2 spawned after Jbird kills a player, and 2 on spawn.
```
Comment (Gboster-0) [inline code/modules/mob/living/simple_animal/abnormality/waw/judgement_bird.dm:125]: inconsistent comment:
the PR says it spawns 3 when breached and 2 when J-bird claims a life
the comment says it spawns 2 when breached and 2 when J-bird claims a life

File: code/modules/mob/living/simple_animal/abnormality/waw/judgement_bird.dm [hunk 2]
```diff
@@ -103,3 +112,73 @@
...
+	V = new(get_turf(src))
+	birdlist+=V
+	V = new(get_turf(src))
+	birdlist+=V
+
+//Kill all burds
+//Burd down
+/mob/living/simple_animal/hostile/abnormality/judgement_bird/death(gibbed)
+	for(var/mob/living/V in birdlist)
+		death()
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/waw/judgement_bird.dm:132]:
```suggestion
		V.death()
```

File: code/modules/mob/living/simple_animal/abnormality/waw/judgement_bird.dm [hunk 3]
```diff
@@ -103,3 +112,73 @@
...
+	if(CheckCombat())
+		judgement_damage = 65
+		return
+
+	var/mob/living/simple_animal/hostile/runawaybird/V = new(get_turf(src))
+	birdlist+=V
+	V = new(get_turf(src))
+	birdlist+=V
+	V = new(get_turf(src))
+	birdlist+=V
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/waw/judgement_bird.dm:126]: Do a `for` loop instead; You also don't need a "V" variable at all.

Comment (Kitsunemitsu) [reply code/modules/mob/living/simple_animal/abnormality/waw/judgement_bird.dm:126]: For loops are technically slower.......

**Review by LanceSmites328 — APPROVED** [review]

**Pull request merged**